pkgs/ffigen/lib/src/header_parser/parser.dart

```diff
@@ -113,6 +113,21 @@ List<Binding> parseToBindings() {
     tuList.add(tu);
   }
 
+  if (hasSourceErrors) {
+    _logger.warning(
+        "The compiler found some warnings/errors in source files. This might generate invalid bindings due to a wrong compiler guess.");
+    if (config.ignoreSourceErrors) {
+      _logger.warning(
+          "Ignored source errors. (User supplied --ignore-source-errors)");
+    } else if (config.language == Language.objc) {
+      _logger.warning("Ignored source errors. (ObjC)");
+    } else {
+      _logger.severe(
+          "Skipped generating bindings due to errors in source files. Either resolve or ignore them (Set --ignore-source-errors on cmd or ignore-source-errors:true in config.");
```

Comment:
I'd not provide documentation on how to skip them, but rather documentation on how to fix errors.

Should we add a wiki page or a markdown file somewhere to document the most common types of errors and their fixes? (What compiler opts to pass etc.)

Reply:
We can, if we can find more atleast a few more (other than missing include paths) a wiki might be worth it.

Would a link to this work for now? https://github.com/dart-lang/native/tree/main/pkgs/ffigen#what-are-these-logs-generated-by-ffigen-and-how-to-fix-them

Reply:
I've added an errors.md file and added reference link to it for now.